ci/coq-tests.el

```diff
@@ -413,6 +422,13 @@ For example, COMMENT could be (*test-definition*)"
 ;; (coq-should-buffer-regexp (regexp-quote "The command has indeed failed with message: Tactic failure: Cannot solve this goal.") "*response*")
 
 (ert-deftest 091_coq-test-regression-Fail()
+  ;; XXX What is the difference between this test and
```

Comment:
The point in coq file is not the same: "(*FailNoTrace*)" vs "(*FailTrace*)". Concretelty the difference is the command `Local Unset Ltac Backtrace.`.

Reply:
OK, it would be nice if the documentation in that file would describe what is tested there.